audio-echo/app/src/main/java/com/google/sample/echo/MainActivity.java

```diff
@@ -38,6 +46,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        //get run-time permission...
+        requestResourcePermission();
```

Comment:
I will study the sample to make things inside onCreate. Mainly to code is to demo the audio audio on Native code, simplify the logic on Java side would be nice